cibuildwheel/architecture.py

```diff
@@ -63,6 +64,12 @@ class Architecture(StrEnum):
     # WebAssembly
     wasm32 = auto()
 
+    # iOS "multiarch" architectures that include both
+    # the CPU architecture and the ABI.
+    arm64_device = "arm64-iphoneos"
+    arm64_simulator = "arm64-iphonesimulator"
+    x86_64_simulator = "x86_64-iphonesimulator"
```

Comment:
Clearly Apple is lugging around some technical debt here, but I think it's probably on my brain if we don't have two different names for these archs.

```suggestion
    arm64_iphoneos = "arm64-iphoneos"
    arm64_iphonesimulator = "arm64-iphonesimulator"
    x86_64_iphonesimulator = "x86_64-iphonesimulator"
```